src/app/hooks/useSearchMovies.ts

```diff
@@ -0,0 +1,11 @@
+import { Movie } from '../../lib/types';
+import useFetch from './useFetch';
+
+export default function useSearchMovies(query: string): {
+  data: Movie[] | null;
+  errorMessage: string | null;
+  isLoading: boolean;
+  refetch: () => void;
```

Comment:
Please reconsider your naming and what needs to be returned. I don't think you will need the refetch.